js/lib/ilib.js

```diff
@@ -129,7 +129,7 @@ ilib._getPlatform = function () {
     		}
     	} catch (e) {}
     	
-        if (typeof(process) !== 'undefined' && typeof(window) === 'undefined' && typeof(module) !== 'undefined') {
```

Comment:
When I ran a unittest on nodejs, I didn't get any fail cases in regards to that change. Could you tell me a your test environment?